src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -1629,6 +1630,45 @@ internal virtual void AggregateRenderPackages(IEnumerable<HelixRenderPackage> pa
 
             lock (element3DDictionaryMutex)
             {
+                // Instancing
+                var pkgs = packages.ToList();
+                if (pkgs.Count > 1)
+                {
+                    var p = pkgs[0];
+                    if (p.Mesh != null && p.Mesh.Positions.Any())
+                    {
+                        var baseId = p.Description;
+                        if (baseId.IndexOf(":", StringComparison.Ordinal) > 0)
+                        {
+                            baseId = baseId.Split(':')[0];
+                        }
+                        var id = ((p.RequiresPerVertexColoration || p.Colors != null || p.RequiresCustomTransform)
```

Comment:
does this somehow conflict with line this line below:
` id = ((rp.RequiresCustomTransform) ? rp.Description : baseId) + PointsKey;`